pkg/interop/native/std/std.go

```diff
@@ -121,3 +121,28 @@ func MemoryCompare(s1, s2 []byte) int {
 	return contract.Call(interop.Hash160(Hash), "memoryCompare", contract.NoneFlag,
 		s1, s2).(int)
 }
+
+// MemorySearch returns index of the first occurence of val in mem.
+// If not found, -1 is returned.
+// It uses `memorySearch` method of StdLib native contract.
+func MemorySearch(mem, pattern []byte) int {
+	return contract.Call(interop.Hash160(Hash), "memorySearch", contract.NoneFlag,
+		mem, pattern).(int)
+}
+
+// MemorySearchIndex returns index of the first occurence of val in mem starting from start.
+// If not found, -1 is returned.
+// It uses `memorySearch` method of StdLib native contract.
+func MemorySearchIndex(mem, pattern []byte, start int) int {
+	return contract.Call(interop.Hash160(Hash), "memorySearch", contract.NoneFlag,
+		mem, pattern, start).(int)
+}
+
+// MemorySearchFull returns index of the first occurence of val in mem starting from start.
+// If backward is true, search backwards, start index is not included.
+// If not found, -1 is returned.
+// It uses `memorySearch` method of StdLib native contract.
+func MemorySearchFull(mem, pattern []byte, start int, backward bool) int {
```

Comment:
Same thing here, we can do `MemorySearchBackward` and pass `true`, other variants are implicitly `false` anyway.